### PR TITLE
Initialize HTML elem's with DOM

### PR DESCRIPTION
**Pull request by mayamolodec**
This commit moves initialisation of HTML elements of the quiz-card
into the main.js. It also excludes the link to Bootstrap (commented) therefore 
there are changes in the style.css, concerning radio input display.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,44 +9,14 @@
     <link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:wght@300&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style/style.css">
     <title>Document</title>
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+    <!-- <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous"> -->
```
Comment (w1n732mu73) [inline index.html]: Don't comment unused things - just remove them

File: scripts/main.js [hunk 2]
```diff
@@ -35,51 +37,76 @@ const questions = [{
...
+}
 
 
-    // Picture
-    const pictureTemplate = `<img class ="illustration" id ="illustration" src = "assets/%illustration%.png">`
-    const pictureHTML = pictureTemplate.replace('%illustration%', 'illustr_'+ question_index);
-    console.log(pictureHTML);
-    picContainer.innerHTML = pictureHTML;
+function showQuestion(){
+    cardImage.src = questions[question_index]['img_link'];
```
Comment (w1n732mu73) [inline scripts/main.js]: Better if you initialize a question at the beginning of a function
```
 const question = questions[questions_index]
 cardImage.src = question.imgLink; 
 ...
```

File: scripts/main.js [hunk 3]
```diff
@@ -35,51 +37,76 @@ const questions = [{
...
+    questionText.textContent = questions[question_index]['question'];
+    cardHeader.append(questionText);
 
     let answerNumber = 1;
+    
+    cardHeader.after(answersList);
 
     // Answers
-    for (answerOption of questions[question_index]['answers']){
+    for (let answerText of questions[question_index]['answers']){
```
Comment (w1n732mu73) [inline scripts/main.js]: Use forEach here

`
 question.answers.forEach((answer) => {...})
`

**Review by w1n732mu73 — CHANGES_REQUESTED** [review]
Good. Please don't merge this PR until you make fixes

File: scripts/main.js (hunk 2 above)
Comment (mayamolodec) [reply scripts/main.js]: Should I git commit --ammend these changes to the current commit, 
or it's better to make a new one, with resolved problems?

Comment (w1n732mu73) [reply scripts/main.js]: It'l be nice if you use amend

**Review by w1n732mu73 — APPROVED** [review]

**Pull request merged**